### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by BuckarooBanzay** [mt-mods/technic#53]
There are already some docs in the upstream wiki, but some of the local customizations
should have documentation too:

* [ ] HV Reactor digilines
* [ ] HV Digilines cable
* [ ] (other stuff i forgot)

A simple markdown documentation should suffice

**Comment by S-S-X** [comment]
Linked to 2.0 release milestone and added API changes to documentation task list.

**Comment by S-S-X** [comment]
I think it is better to keep deprecated stuff (that is still available) away from documentation, and remove if present.
If still somehow relevant like deprecated function or table moved to `compat/whteveritwas.lua` then include deprecation warnings using Minetest logging facilities.

If something should be removed it might be good to consider moving it to compat directory and throwing clean `error(message)` about situation instead of just removing said function.

(not sure if we want to do that but this also allows `I_do_not _care_about_deprecation = true` configuration for some stuff).

**Comment by S-S-X** [comment]
> There are already some docs in the upstream wiki

This documentation is already pretty much irrelevant and becomes even less relevant all the time because so much has changed, probably just confusing:
* Whole network and switching works differently
* Registrations for tools / machines / cables are about change
* Some machines work different way and have different features
* API and especially internal working is generally very different
* Commands are different
* I am sure there's a lot more that could be added to this list

**Comment by OgelGames** [comment]
Yeah, I had a look at the wiki, and there is nothing of use there, and one page is just plain incorrect (for both versions): https://github.com/minetest-mods/technic/wiki/Chests

**Comment by mase76** [comment]
I don't get any answer, not even an event when sending commands to the chest. Can you give a small example for the count_item command?

**Comment by S-S-X** [comment]
> I don't get any answer, not even an event when sending commands to the chest. Can you give a small example for the count_item command?

```lua
{
  command = "count_item",
  item = "default:dirt"
}
```

Here's code handling commands: https://github.com/mt-mods/technic/blob/b4cde134128091bb71e3d056f51814f64bfa79cb/technic_chests/digilines.lua#L25-L102
